src/encoded/static/components/globals.js

```diff
@@ -313,6 +313,33 @@ export const biosampleTypeList = [
     'technical sample',
 ];
 
+
+// order is important
+export const DeeplyProfiledCellLineList = [
+    'A549',
+    'A673',
+    'Caco-2',
+    'Calu3',
+    'GM12878',
+    'GM23338',
+    'H1',
+    'H9',
+    'HCT-116',
+    'HEPG2',
+    'HFFc6',
+    'HMEC',
+    'HUVEC',
+    'IMR-90',
+    'K562',
+    'MCF-7',
+    'MCF10A',
+    'OCI-LY7',
+    'PC-9',
+    'PC3',
+    'Panc1',
+    'WTC11',
+];
```

Comment:
Seems like a very specific thing to put into globals.js. Could deeply profiled cell-lines really be shared across disparate parts of the code base? I probably would have put this into matrix.js to share them between the two deeply profiled matrices.